### PR TITLE
Added order lines for Klarna Pay Later payment method

### PR DESCRIPTION
**Pull request by tiutiunnykleasoft**

### CODE REVIEW

**Review by volgustleasoft — COMMENTED** [review]
For Afterpay, it seems, also need to add an order_lines

File: upload/system/library/emshelper.php [hunk 1]
```diff
@@ -638,7 +634,7 @@ public function getOrderLineTaxRate($paymentMethod, $price, $taxClassId)
             }
         }
 
-        return static::formatAmountToCents($taxRate);
+        return (int) round($taxRate * 100);
```
Comment (volgustleasoft) [inline upload/system/library/emshelper.php:642]: Why is it not taking into account currency (without using the getAmountInCents function)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,4 +23,8 @@
 
 ** 1.5.0 **
 
-* Fixed payment URL for Klarna Pay Later
\ No newline at end of file
+* Fixed payment URL for Klarna Pay Later
+
+** 1.5.1 ** 
+
+* Added order lines for Klarna Pay Later payment method
```
Comment (volgustleasoft) [inline CHANGELOG.md]: Expand Changelog information.

**Review by volgustleasoft — COMMENTED** [review]
Change plugin version in helper - const PLUGIN_VERSION = '1.5.0';

File: upload/system/library/emshelper.php (hunk 1 above)
Comment (tiutiunnykleasoft) [reply upload/system/library/emshelper.php:642]: Because formatAmountToCents actually working incorrect and return mismatched value

File: upload/system/library/emshelper.php [hunk 3]
```diff
@@ -275,10 +268,10 @@ public function getOrderData(array $orderInfo, $paymentMethod)
         $issuerId = array_key_exists('issuer_id', $paymentMethod->request->post)
             ? $paymentMethod->request->post['issuer_id'] : null;
 
-        $total_amount_in_cents = $this->getAmountInCents($orderInfo, $paymentMethod->currency);
+        $total_amount = self::getAmountInCents($orderInfo,$paymentMethod->currency);
```
Comment (volgustleasoft) [inline upload/system/library/emshelper.php]: Function getAmountInCents is not static.

File: upload/system/library/emshelper.php [hunk 4]
```diff
@@ -638,7 +642,7 @@ public function getOrderLineTaxRate($paymentMethod, $price, $taxClassId)
             }
         }
 
-        return static::formatAmountToCents($taxRate);
+        return (int) $taxRate * 100;
```
Comment (volgustleasoft) [inline upload/system/library/emshelper.php]: Why did you remove rounding?

**Review by volgustleasoft — COMMENTED** [review]
Comments not related to changes to this PR
1. 'locale' => self::formatLocale($orderInfo['language_code']) - function is not static
2. return sprintf('OpenCart v%s', self::PLUGIN_VERSION); - I propose to replace it with static for uniformity

File: upload/system/library/emshelper.php [hunk 5]
```diff
@@ -427,7 +424,7 @@ public function loadCallbackFunction($paymentMethod)
             if ($emsOrder['status'] == 'completed') {
                 $paymentMethod->response->redirect($this->getSucceedUrl($paymentMethod, $orderInfo['order_id']));
             } elseif ($emsOrder['status'] == 'processing' || $emsOrder['status'] == 'new') {
-                $paymentMethod->response->redirect($paymentMethod->emsHelper->getProcessingUrl($paymentMethod));
+                $paymentMethod->response->redirect($this->getProcessingUrl($paymentMethod));
```
Comment (volgustleasoft) [inline upload/system/library/emshelper.php:427]: There is also such a call to internal functions in the helper:
$paymentMethod->emsHelper->getOrderStatus
